src/api/Recents.js

```diff
@@ -11,7 +11,16 @@ import Base from './Base';
 import FileAPI from './File';
 import FolderAPI from './Folder';
 import WebLinkAPI from './WebLink';
-import { DEFAULT_ROOT, CACHE_PREFIX_RECENTS, FIELD_DATE, SORT_DESC, ERROR_CODE_FETCH_RECENTS } from '../constants';
+import {
+    DEFAULT_ROOT,
+    CACHE_PREFIX_RECENTS,
+    ERROR_CODE_FETCH_RECENTS,
+    FIELD_DATE,
+    FIELD_REPRESENTATIONS,
+    GRID_VIEW_DEFAULT_JPG_DIMENSIONS,
```

Comment:
We can probably name these something non-component related, like `X_REP_HINT_DIMENSIONS_DEFAULT`. We could also put the entire string into another constant that consumes the first, rather than duplicating it in each API.

Do we also want to add `paged=false`, similar to [box-content-preview](https://github.com/box/box-content-preview/blob/master/src/lib/constants.js#L96)? We only need the first page, I think.